README.md

```diff
@@ -139,3 +137,6 @@ get involved:
     your changes, and submit a [Pull
     Request](https://github.com/codee-com/open-catalog/pulls). We'd love to see
     your contributions!
+
+
+[^1]: [Codee](https://www.codee.com)
```

Comment:
I wouldn't move this to the end of the page. What do you think?

Reply:
This way is closest to how GitHub and docusaurus render it.

Also, footnotes are sopposed to be and the end of the page, right? :laughing:

Reply:
> This way is closest to how GitHub and docusaurus render it.

I see. Thank you so much for the clarification!

> Also, footnotes are sopposed to be and the end of the page, right? 😆

😆